build.sbt

```diff
@@ -16,5 +16,10 @@ lazy val root = (project in file("."))
     )
   )
 libraryDependencies ++= Seq(
-  "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % "test"
+  "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % "test",
+  "org.typelevel" %% "cats-effect" % "3.3.14",
+  "org.tpolecat" %% "doobie-core"      % "1.0.0-RC1",
+  "org.tpolecat" %% "doobie-postgres"  % "1.0.0-RC1",
+  "org.tpolecat" %% "doobie-scalatest" % "1.0.0-RC1" % "test"
+  , evolutions, jdbc
```

Comment:
```suggestion
  "org.tpolecat" %% "doobie-scalatest" % "1.0.0-RC1" % "test",
  evolutions, 
  jdbc
```

Reply:
Ah, I did not notice that, thank you, will change it!